### PR TITLE
Resolve w3c

### PR DESCRIPTION
**Pull request by rodolfomiranda**
Adding a feature to resolve DID documents transformed in a way that are W3C conformant. 
The transformation basically put the public keys in a `verificationMethod` property array with correct decorators and public keys encoded as `publicKeyBase58`. 
The cli command is the same 'resolve-did-prism` command but adding --w3c flag.

Note the TODO comment in  line 266

### CODE REVIEW

**Comment by 2byrds** [comment]
Awesome!

File: src/main/kotlin/Command.kt [hunk 1]
```diff
@@ -236,12 +240,54 @@ class ShowDID : Subcommand("show-did", "Show DID document") {
...
     private val did by argument(ArgType.String, "did", "PRISM did (canonical or long form)")
+    private val w3c by option(ArgType.Boolean, "w3c", "w3c", "W3C compliant DID document").default(false)
+
     @OptIn(PrismSdkInternal::class, ExperimentalProtoJson::class)
     override fun execute() {
         val dataModel = getDidDocument(did)
         println(green("-- $name --"))
         println("DID document")
-        println(dataModel.didData.toProto().encodeToJsonString())
+        if (w3c) {
```
Comment (Essbante) [inline src/main/kotlin/Command.kt:245]: move this into a function in wal-lib/DLT.kt file

**Review by Essbante — CHANGES_REQUESTED** [review]
Hi @rodolfomiranda, thanks for your contribution!
Command.kt classes are intended only for wallet object storage management and CLI interface definition. Functions in wal-lib handle the logic. 

Please make the following change to keep the code base organized.
- move the logic into a function in wal-lib (DLT.kt file) and call it in the "if(w3c)" on the command definition.
- Make the pull request against develop branch instead of the main branch. 

This way, other implementations can use the w3c did resolver by importing wal-lib and not only with wal-cli.

Also, if you haven't, please use ktlint to format the code before submitting it. There is a nice Ktlint plugin available for IntelliJ.

File: src/main/kotlin/Command.kt (hunk 1 above)
Comment (2byrds) [reply src/main/kotlin/Command.kt:245]: Great feedback!

**Comment by rodolfomiranda** [comment]
Changed PR to `develop` branch and moved w3c function to was-library call.

**Review by Essbante — APPROVED** [review]

**Pull request merged**